submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -9,185 +9,144 @@
   </head>
   <body>
     <nav>
-      <div class="nav-container">
-        <div class="nav-el nav-el-text" tabindex="0">
-          <p>Mail</p>
-        </div>
-        <div class="nav-el nav-el-text" tabindex="0">
-          <p>Images</p>
-        </div>
-
-        <!-- #wrap-el - обертка для input + label + popup.
-              CSS к #wrap-el не применял -->
-        <div id="wrap-el">
-
-          <input type="checkbox" id="button-1" tabindex="0">
-          <div class="nav-el" id="nal-el-menu">
+      <ul class="nav-container">
+        <li class="nav-el">
+          <a href="#" tabindex="0">Mail</a>
+        </li>
+        <li class="nav-el">
+          <a href="#" tabindex="0">Images</a>
+        </li>
+
+        <li class="nav-el">
+          <input type="checkbox" id="button-1"  tabindex="0">
+          <a href="#" tabindex="-1">
             <label for="button-1" id="menu-label">
-                <img src="img/apps.jpg" alt="Menu icon" />
+              <img src="img/apps_new.png" alt="Menu icon">
             </label>
-          </div>
-<!-- закончил рефакторинг тут -->
-          <section id="apps-window">
-            <div id="top-triangle"></div>
-            <div id="top-triangle-border"></div>
+          </a>
 
+          <div id="apps-window">
             <input type="checkbox" id="button-2" tabindex="2">
-            <div id="more">
-              <label for="button-2">
-                <div id="more-label-div">More</div>
+            <button id="more" tabindex="-1">
```

Comment:
You can't nest one interactive element inside another one, don't do it in the future)
All interactive elements(as `<button>, <a>, <input/>, etc.`) by default has focus, no need to use `tabindex` attribute for them, except situation when you want to change your default order of elements.

Reply:
Thank you, Alexandr.
I've fixed it in my updated code.